vms/platformvm/blocks/executor/verifier.go

```diff
@@ -265,15 +265,15 @@ func (v *verifier) banffNonOptionBlock(b blocks.BanffBlock) error {
 		)
 	}
 
-	nextStakerChangeTime, err := executor.GetNextStakerChangeTime(parentState)
```

Comment:
I'm not a friend in renaming, if we overwrite builderf.go, these changes can be omitted